### PR TITLE
Add links to pecl and git repositories

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: docs/event-loop/README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ The event loop should be accessed through the methods provided by `Amp\Loop`. On
...
-| Class                     | Extension                                              |
-| ------------------------- | ------------------------------------------------------ |
-| `Amp\Loop\NativeDriver`     | –                                                    |
-| `Amp\Loop\EvDriver`         | [`pecl/ev`](https://pecl.php.net/package/ev)             |
-| `Amp\Loop\EventDriver`      | [`pecl/event`](https://pecl.php.net/package/event) |
-| `Amp\Loop\UvDriver`         | [`php-uv`](https://github.com/bwoebi/php-uv)             |
+| Class                     | Extension                                              | Repository |
+| ------------------------- | ------------------------------------------------------ | ---------- |
+| `Amp\Loop\NativeDriver`   | –                                                      | -          |
+| `Amp\Loop\EvDriver`       | [`pecl/ev`](https://pecl.php.net/package/ev)           | [`php-ev`](https://github.com/lvht/php-ev) |
```
Comment (kelunik) [inline docs/event-loop/README.md]: That's the wrong link, should be https://bitbucket.org/osmanov/pecl-ev.

Comment (enumag) [reply docs/event-loop/README.md]:
```suggestion
| `Amp\Loop\EvDriver`       | [`pecl/ev`](https://pecl.php.net/package/ev)           | [`php-ev`](https://bitbucket.org/osmanov/pecl-ev) |
```

**Comment by enumag** [comment]
Sorry, forgot about this... link changed.

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**